src/client/datascience/commands/commandRegistry.ts

```diff
@@ -487,6 +488,10 @@ export class CommandRegistry implements IDisposable {
         env.openExternal(Uri.parse(`https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter`));
     }
 
+    // Open up our variable viewer using the command that VS Code provides for this
```

Comment:
The connection between 'jupyterViewVariables.focus' and the actual webview opening is hard to find. Could you add some more comments here explaining it?

Reply:
Yeah, I actually found that command hard to find. I'll add a comment into the code here.